libsolc/libsolc.cpp

```diff
@@ -31,7 +31,7 @@
 #include <list>
 #include <string>
 
-#include "license.h"
```

Comment:
This is set in cmake in a different way than the other include paths, I fear it has to stay like this. At least it might be that this causes problems with ppa and I don't want to risk it.